app/uk/gov/hmrc/helptosavefrontend/views/register/account_created.scala.html

```diff
@@ -48,51 +51,73 @@
 @layout(title) {
 
 @if(hasErrors) {
-@errorSummary(form.errors)
+    @viewHelpers.govukErrorSummary(ErrorSummary(errorList =
+    form.errors.map { error =>
+        ErrorLink(
+        href = Some(s"#${error.key}"),
+        content = Text(messages("hts.payInNow.error"))
+        )
+    },
+    title = Text(messages("hts.global.error-summary.title"))))
 }
-
-  <div class="govuk-box-highlight">
-    <h1 class="govuk-heading--xl">@title</h1>
-    <span class="heading-large">
+  <div class="govuk-panel govuk-panel--confirmation">
+    <h1 class="govuk-panel__title">@title</h1>
+    <div class="govuk-panel__body">
       @Messages("hts.account-created.title.p1")
       <strong>@accountNumber</strong>
-    </span>
+    </div>
   </div>
 
   <p>@Html(Messages("hts.register.create_account.confirmation-info.p1")) @email@Html(Messages("hts.register.create_account.confirmation-info.p2"))</p>
 
-<div class="form-group @if(hasErrors){form-group-error}">
-  <h2 class="heading-medium">@Messages("hts.register.create_account.what-you-need-to-do.title")</h2>
+<div class="govuk-form-group @if(hasErrors){govuk-form-group--error}">
+  <h2 class="govuk-heading--m">@Messages("hts.register.create_account.what-you-need-to-do.title")</h2>
     <p>@Messages("hts.register.create_account.what-you-need-to-do.p1")</p>
```

Comment:
all paragraphs should, by default, use the class `govuk-body` this ensures spacing and line heights etc are consistent